src/sagemaker_pytorch_serving_container/default_inference_handler.py

```diff
@@ -12,12 +12,24 @@
 # language governing permissions and limitations under the License.
 from __future__ import absolute_import
 
+import logging
+import os
+import sys
 import textwrap
 
 import torch
-
 from sagemaker_inference import content_types, decoder, default_inference_handler, encoder
 
+INFERENCE_ACCELERATOR_PRESENT_ENV = 'SAGEMAKER_INFERENCE_ACCELERATOR_PRESENT'
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+logger.addHandler(logging.StreamHandler(sys.stdout))
+
+
+class FileNotFoundError(OSError):
```

Comment:
Can we use the built-in exception: https://docs.python.org/3.6/library/exceptions.html#FileNotFoundError instead?

Reply:
removed the class and using built-in exceptions